src/txn/txn.c

```diff
@@ -1484,7 +1484,7 @@ __txn_mod_sortable_key(WT_SESSION_IMPL *session, WT_TXN_OP *opt)
     case (WT_TXN_OP_INMEM_ROW):
         return (true);
     }
-    WT_ASSERT_ALWAYS(session, false, "Unhandled op type encountered.");
+    return (-1);
```

Comment:
This isn't a good idea, while return (true) should be a positive integer technically true is anything on zero. So by returning -1 you're technically returning true.

Reply:
Your comment about anything other than zero evaluating to "True" is correct and is a detail I overlooked. We've since changed up the design to make clang happy and Windows compile here [dfaf25a](https://github.com/wiredtiger/wiredtiger/pull/9453/commits/dfaf25ae78042edd670125098360328b89bf789b)